<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cookGUI</class>
 <widget class="QMainWindow" name="cookGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#cookGUI{
background-color: rgb(0, 0, 0);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="orderSlot1" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#orderSlot1{
border: 2px solid;
border-color: rgb(0, 170, 0);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="orderArea1" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 2px solid green;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 170, 0);</string>
               </property>
               <property name="text">
                <string>Order: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="orderNum1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 170, 0);</string>
               </property>
               <property name="text">
                <string>00</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Arial&quot;;
border-color: rgb(0, 0, 0);
color: rgb(0, 170, 0);
background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="doneButton1">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 170, 0);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>DONE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="orderSlot2" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#orderSlot2{
border: 2px solid;
border-color: rgb(0, 170, 0);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QWidget" name="orderArea2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 2px solid green;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 170, 0);</string>
               </property>
               <property name="text">
                <string>Order: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="orderNum2">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 170, 0);</string>
               </property>
               <property name="text">
                <string>00</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_9" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListWidget" name="listWidget_2">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Arial&quot;;
border-color: rgb(0, 0, 0);
color: rgb(0, 170, 0);
background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="doneButton2">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 170, 0);
font: 20pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>DONE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="orderSlot3" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#orderSlot3{
border: 2px solid;
border-color: rgb(0, 170, 0);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QWidget" name="orderArea3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 2px solid green;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 170, 0);</string>
               </property>
               <property name="text">
                <string>Order: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="orderNum3">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 170, 0);</string>
               </property>
               <property name="text">
                <string>00</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_10" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="listWidget_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Arial&quot;;
border-color: rgb(0, 0, 0);
color: rgb(0, 170, 0);
background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="doneButton3">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 170, 0);
font:20pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>DONE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="orderSlot4" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#orderSlot4{
border: 2px solid;
border-color: rgb(0, 170, 0);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QWidget" name="orderArea4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 2px solid green;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 170, 0);</string>
               </property>
               <property name="text">
                <string>Order: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_8">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="orderNum4">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 170, 0);</string>
               </property>
               <property name="text">
                <string>00</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_11" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListWidget" name="listWidget_4">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Arial&quot;;
border-color: rgb(0, 0, 0);
color: rgb(0, 170, 0);
background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="doneButton4">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 170, 0);
font: 20pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>DONE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
